en/test_and_ci/integration_testing_mavsdk.md

```diff
@@ -33,4 +33,43 @@ To see all possible command line arguments, check out:
 
 ```sh
 test/mavsdk_tests/mavsdk_test_runner.py -h
+
+usage: mavsdk_test_runner.py [-h] [--log-dir LOG_DIR] [--speed-factor SPEED_FACTOR] [--iterations ITERATIONS] [--abort-early] [--gui] [--model MODEL]
+                             [--case CASE] [--debugger DEBUGGER] [--verbose]
+                             config_file
+
+positional arguments:
+  config_file           JSON config file to use
+
+optional arguments:
+  -h, --help            show this help message and exit
```

Comment:
FYI only, the only inconsistent capitalisation is the S in show.

Reply:
Aha, this means I need to not capitalize all of them because `--help` is added automatically by argparse.

Reply:
https://github.com/PX4/Firmware/pull/14421